src/dash-table/components/Export/index.tsx

```diff
@@ -10,12 +10,14 @@ interface IExportButtonProps {
     virtual_data: IDerivedData;
     visibleColumns: Columns;
     export_headers: ExportHeaders;
+    export_className?: string;
+    export_text?: string;
     merge_duplicate_headers: boolean;
 }
 
 export default React.memo((props: IExportButtonProps) => {
 
-    const { columns, export_columns, export_format, virtual_data, export_headers, visibleColumns, merge_duplicate_headers } = props;
+    const { columns, export_columns, export_format, virtual_data, export_headers, export_className='export', export_text='Export', visibleColumns, merge_duplicate_headers } = props;
```

Comment:
It's better to put these defaults in [`defaultProps`](https://github.com/plotly/dash-table/blob/570912fa59485b3964f68ae26b8766a9fcf2b97a/src/dash-table/dash/DataTable.js#L34) (and then NOT explicitly include them in the prop docstring) as that's less duplication and also makes them available to Python.